Cache/IO.lean

```diff
@@ -367,7 +367,8 @@ def unpackCache (hashMap : HashMap) (force : Bool) : CacheM Unit := do
     stdin.putStr <| Lean.Json.compress <| .arr config
     let exitCode ← child.wait
     if exitCode != 0 then throw <| IO.userError s!"leantar failed with error code {exitCode}"
-    IO.println s!"unpacked in {(← IO.monoMsNow) - now} ms"
+    IO.println s!"Unpacked in {(← IO.monoMsNow) - now} ms"
+    IO.println s!"Completed successfully!"
```

Comment:
Since you're just printing a simple string, you don't need the `s!` macro.
```suggestion
    IO.println "Completed successfully!"
```
In fact, I wonder if it's nicer to combine these two lines, as in `IO.println s!"Unpacked in {(← IO.monoMsNow) - now} ms\nCompleted successfully!"`.

On a higher level: is this the best place to put this message? I would rather expect this line to be wherever the main logic of the `cache get` command is... (so, if some other step errors, you never get a "success" message from this later step, by design).

Reply:
Thank you for the comments!
I assumed that errors cause the function to halt, then `unpackCache` won't be called. I'm not sure how the existence of errors is being checked, for instance at `getFiles` in `Cache/Requests.lean` or `main` in `main.lean`. Do  you have any hints?

Reply:
Good question! I don't know myself, so I took a look at the code.
The two previous functions exit the process (with exit code 1) if there's any error, so your assumption seems correct.
I added some documentation to this effect, and also a success message if only decompression was wanted.